dockstore-webservice/src/main/java/io/dockstore/webservice/core/OrganisationUser.java

```diff
@@ -0,0 +1,173 @@
+package io.dockstore.webservice.core;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@Table(name = "OrganisationUser")
```

Comment:
since this is really a join table, name it like the others (entry_label, user_entry, etc.)